libs/domain-service/src/resolvers/index.ts

```diff
@@ -31,7 +31,7 @@ export const resolveDomain = async (
 
   const responses = Promise.allSettled(
     registries.map((registry) =>
-      network<string>({
+      axios.request<string>({
```

Comment:
Because using the `network` module that is coming from `live-common` (for now, it could be changed once it becomes its own lib) so it's creating a cycle dependency once `@ledgerhq/domain-service` is added to `hw-app-eth`:

`@ledgerhq/live-common` —dep—> `hw-app-eth` —dep—> `@ledgerhq/live-common`